host/sgx/sgxsign.c

```diff
@@ -645,6 +647,22 @@ static oe_result_t _init_sigstruct(
         mrenclave,
         sizeof(*mrenclave)));
 
+    if (attributes & SGX_FLAGS_KSS)
+    {
+        if (isv_family_id)
+            OE_CHECK(oe_memcpy_s(
+                sigstruct->isvfamilyid,
+                sizeof(sigstruct->isvfamilyid),
+                isv_family_id,
+                sizeof(sigstruct->isvfamilyid)));
```

Comment:
isv_family_id is pointer to uint8 instead of struct so its size would not work.

Reply:
> isv_family_id is pointer to uint8 instead of struct so its size would not work.

sizeof(*isv_family_id) will return the size of what it points to.

Reply:
Done